src/guide/testing.md

```diff
@@ -86,7 +86,7 @@ Its guiding principle is that the more tests resemble the way software is used,
 
 #### Vue Test Utils
 
-Vue Test Utils is the low-level component testing library that was written to provide users access to Vue specific APIs. While it is officially maintained, we recommend most users use Vue Testing Library, which is an abstraction over Vue Test Utils. 
+Vue Test Utils is the official low-level component testing library that was written to provide users access to Vue specific APIs. If you are starting testing Vue applications, we would recommend using Vue Testing Library, which is an abstraction over Vue Test Utils. 
```

Comment:
Should this be "If you are new to testing" or "If you are starting a new application"?

I like the testing library philosophy more, but I like the VTU API more 😱 that said I don't mind which library is used, as long as people are testing their apps 👍

Reply:
fixed! thanks. copy&paste went wrong 😇